doc/build-windows.md

```diff
@@ -66,12 +69,32 @@ See also: [dependencies.md](dependencies.md).
 
 ## Building for 64-bit Windows
 
-To build executables for Windows 64-bit, install the following dependencies:
+THe first step is to install the mingw-w64 cross-compilation tool chain. Due to different Ubuntu
+packages for each distribution and problems with the Xenial packages the steps for each are different.
+
+Ubuntu Trusty 14.04:
+
+	sudo apt-get install g++-mingw-w64-x86-64 mingw-w64-x86-64-dev
+
+Ubuntu Xenial 16.04 and Windows Subsystem for Linux (*)(**):
```

Comment:
Use footnotes (`[^1]`) rather than `*` because `*` is used for formatting. (Same in other places).